apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -112,18 +124,20 @@ class MultipleChoiceOverviewTable extends Component {
   );
 
   getColumns = (sortable) => {
-    const maxAnswerChoicesLength = this.props.questionAnswerData.reduce((answersTotal, currentAnswerCount) => {
-      return Math.max(answersTotal, currentAnswerCount.answers.length);
-    }, 0);
-
     let dataColumns = [];
```

Comment:
I'm not sure I totally understand the reason for the re-factoring in this function. 
Your new code creates an empty `dataColumns`, pushes one element to it, and then deconstructs it to add it to an array. Would it be simpler to just add `this.getQuestionColumn(sortable)` to your array like this?

```
return [
  this.getQuestionColumn(sortable),
  ...columnLabelNames,
  this.getNotAnsweredColumn()},
];
```
I think it could make your final constructed array easier to read.
Note:  you could make `this.getNotAnsweredColumn()` return an object that already has the property `NOT_ANSWERED` so you don't have to reconstruct it here.

Reply:
Thanks, it does make it easier to read.  Fixed.